modules/services/gpg-agent.nix

```diff
@@ -129,26 +129,6 @@ in
           configuration file.
         '';
       };
-
-      pinentryFlavor = mkOption {
```

Comment:
how that relates to the change?

Reply:
It's not at all related, this change got included by mistake. Thanks for noticing!